### DISCUSSION TITLE
Film Score Capability and Desperate Needs

### DISCUSSION THREAD

**Discussion by cfirwin3** [musescore/MuseScore#20569] [Ideas]
I demonstrate what Musescore 4 can do for film score (it's pretty great)... and I make an earnest (and very direct) plea for attention on a long-overdue critical feature that has been lost in regression, midi timecode transport sync:

https://youtu.be/yTBQE1Gyv1w?si=sjqVXs6rUNwJXyEZ

-Carl

**Comment by shoogle** [comment]
JACK support is being worked on by a community member in PR https://github.com/musescore/MuseScore/pull/19246
